packages/backend-plugin-api/api-report.md

```diff
@@ -208,7 +208,7 @@ export namespace coreServices {
 // @public
 export function createBackendModule(
   options: CreateBackendModuleOptions,
-): () => BackendFeature;
+): BackendFeature;
```

Comment:
Would something like this work for now?

```ts
interface Derp extends BackendFeature {
  /** @deprecated nah */
  (): BackendFeature
}
```

Reply:
This is basically what I started in https://github.com/backstage/backstage/blob/62b2dab79c6ea148580e302ece49072cb04d0881/packages/backend-plugin-api/src/types.ts#L30

Inheriting so as not to pollute the original type is very clever.

But I ran into troubles at runtime with this. Haven't found the bug yet. Things end up with duplicate registrations.

Reply:
Hmm, should be possible to roll this out as just a typescript change for now though, no runtime changes other than updated usage

Reply:
Done!